js/main.js

```diff
@@ -1,4 +1,5 @@
 let matsList = ''; // final matList output
+let displayList = ''; // final list for HTML display

```

Comment:
You will want to start moving away from global variables. It generally just makes life easier